sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -125,7 +125,7 @@ rename-mapping:
   SubResourceWithColocationStatus: ComputeSubResourceDataWithColocationStatus
```

Comment:
Dapeng is also working on the `Comtpute`, please make sure merge is correct

Reply:
Offline talked with Dapeng, this PR will only cover all the batch 2 RPs except compute, so I just reverted the change in compute.